compiler/test/fail_compilation/fail163.d

```diff
@@ -80,7 +81,8 @@ void test6()
 /*
 TEST_OUTPUT:
 ---
-fail_compilation/fail163.d(89): Error: cannot implicitly convert expression `& x` of type `const(int)*` to `int*`
+fail_compilation/fail163.d(89): Error: cannot implicitly convert `const(int)*` to `int*`
+fail_compilation/fail163.d(89):        Note: Converting const to mutable requires an explicit cast (`cast(int*)`).
```

Comment:
ditto for the tests following this one, but these are off by two instead of off by one in the above cases